### PR TITLE
Layout examples

### PR DESCRIPTION
**Pull request by rachelandrew**
A bunch of box model examples, plus positioned layout physical offsets.

### CODE REVIEW

File: live-examples/css-examples/basic-box-model/clear.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.example-container {
+    background-color: #eee;
+    border: .75em solid;
+    padding: .75em;
+    text-align: left;
+}
+
+.floated-left {
+    border: solid 10px #FFC129;
```
Comment (wbamberg) [inline live-examples/css-examples/basic-box-model/clear.css]: We should have lowercase for hex values, here and elsewhere in this PR.

File: live-examples/css-examples/basic-box-model/clear.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    background-color: rgba(255, 244, 219, 0.6);
```
Comment (wbamberg) [inline live-examples/css-examples/basic-box-model/clear.css]: We should omit leading zeroes, here and elsewhere in this PR.

File: live-examples/css-examples/basic-box-model/clear.css (hunk 1 above)
Comment (rachelandrew) [reply live-examples/css-examples/basic-box-model/clear.css]: fixed.

File: live-examples/css-examples/basic-box-model/clear.css (hunk 2 above)
Comment (rachelandrew) [reply live-examples/css-examples/basic-box-model/clear.css]: fixed.

**Pull request merged**